qiskit/aqua/algorithms/minimum_eigen_solvers/qaoa/qaoa.py

```diff
@@ -105,7 +110,8 @@ def __init__(self,
                 potentially the expectation values can be computed in parallel. Typically this is
                 possible when a finite difference gradient is used by the optimizer such that
                 multiple points to compute the gradient can be passed and if computed in parallel
-                improve overall execution time.
+                improve overall execution time. Deprecated if a gradient operator or function is
```

Comment:
What do you mean by deprecated? Is it deprecated or just ignored? 🙂

Reply:
Ignored yes. I'll change it right away.